layouts/millennium_r4_sidebar.html

```diff
@@ -8,6 +8,12 @@ <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></
           <li><a href="/millennium/r4/proprietary-codes-and-systems/">Proprietary Codes and Systems</a></li>
         </ul>
       </li>
+      <li class="js-topic">
+        <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></a><a href="/millennium/r4/care-provision/">Care Provision</a></h3>
+        <ul class="js-guides">
+          <li><a href="/millennium/r4/care-provision/service-request/">ServiceRequest</a></li>
+        </ul>
+      </li>
```

Comment:
Let us actually have this after Conformance. Even-though not alphabetical it makes more sense to us that way as conformance carries the information in relation to all the other resources.

Reply:
You can find the changes here: https://github.com/cerner/fhir.cerner.com/pull/389/commits/18a592885426eefd181f05aa020842939e654c6a